includes/admin/settings/class-wcmpbe-settings-data.php

```diff
@@ -192,7 +192,7 @@ static function() use ($section) {
             $optionIdentifier,
             array_replace_recursive(
                 $defaults,
-                get_option($optionIdentifier)
+                (array) get_option($optionIdentifier) ?: []
```

Comment:
dit cast nu altijd het resultaat van get_option naar array, dat zal misschien wel goed gaan, maar dit is wat duidelijker in wat het doet:
```suggestion
                (array) (get_option($optionIdentifier) ?: [])
```